build.gradle

```diff
@@ -1,9 +1,12 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
+            url "https://oss.sonatype.org/content/repositories/snapshots/"
+        }
     }
     dependencies {
-        classpath 'org.javafxports:jfxmobile-plugin:1.0.8'
+        classpath 'org.javafxports:jfxmobile-plugin:1.0.10-SNAPSHOT'
```

Comment:
I think we can wait till 1.0.10 is released before we make this change? (avoiding SNAPSHOT dependencies)